### DISCUSSION TITLE
Slider tooltip format in a code behind

### DISCUSSION THREAD

**Discussion by Switch0621** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2726] [Q&A] [answered]
Hello!

` wpf:SliderAssist.ToolTipFormat = "{0:F1}"`

How can i do this XAML in a code?

**Comment by ElieTaillard** [comment, marked as answer]
You can use the `SetToolTipFormat` function.

Example:
`MaterialDesignThemes.Wpf.SliderAssist.SetToolTipFormat(yourElement, value)`

This is the function in the source code:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/SliderAssist.cs#L28

This function was added in this PR: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/commit/4c0415cac8c72f5fedeff8331da8bcda1f7c70ba

You may not have this feature in the version you are using because it's a recent addition.

**Reply by Switch0621** [reply]
> You can use the `SetToolTipFormat` function.
> 
> Example: `MaterialDesignThemes.Wpf.SliderAssist.SetToolTipFormat(yourElement, value)`
> 
> This is the function in the source code: https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/SliderAssist.cs#L28
> 
> This function was added in this PR: [4c0415c](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/commit/4c0415cac8c72f5fedeff8331da8bcda1f7c70ba)
> 
> You may not have this feature in the version you are using because it's a recent addition.

Thank you so much, Xaalek!

**Reply by ElieTaillard** [reply]
Glad it helped you
